### ISSUE TITLE
Generated YAML can be invalid

### ISSUE THREAD

**Issue by tllilleh** [MichaBrugger/booksidian_plugin#20]
The generated YAML shown below is invalid:
```
---
author: James S.A. Corey
title: Leviathan Wakes
series: Expanse, #1
---
```

The issue is with the `series` field as the # needs to be quoted to be valid YAML.  One simple solution might be to quote all values in `getFrontmatterLines`, or make the default `prefix` and `suffix` quotes.

**Comment by tllilleh** [comment]
I worked around it by adding the quotes in my config, but is seems like something that should happen automatically.

![pic-selected-230127-1728-42](https://user-images.githubusercontent.com/24443261/215224939-a8a74e42-49bb-4be7-9bca-b901b6fb45cd.png)

**Comment by zacharyw** [comment]
@tllilleh Should be resolved as of 3.0.1!
